### PR TITLE
ISPN-11411 Fix ServerEventLoggerTest failures

### PR DESCRIPTION
**Pull request by tristantarrant**
https://issues.redhat.com/browse/ISPN-11411

### CODE REVIEW

**Comment by anistor** [comment]
checkstyle does not approve

File: server/rest/src/test/java/org/infinispan/rest/resources/CacheV2ResourceTest.java [hunk 1]
```diff
@@ -64,9 +64,9 @@ private ConfigurationBuilder getIndexedPersistedCache() {
       return builder;
    }
 
-   @AfterClass
+   @AfterMethod
```
Comment (danberindei) [inline server/rest/src/test/java/org/infinispan/rest/resources/CacheV2ResourceTest.java]: I was going to ask you last night to add a comment on why `@AfterTest` is needed, but GitHub wouldn't allow me to click submit :)

**Comment by anistor** [comment]
LGTM. Thanks @tristantarrant !

**Comment by anistor** [comment]
Backported to 10.1.x also

**Pull request merged**